### DISCUSSION TITLE
Extended user information with `extended=full,vip`

### DISCUSSION THREAD

**Discussion by henrikfroehling** [trakt/trakt-api#403] [Q&A] [answered]
I noticed via Postman that it is possible to retrieve extended user information (`?extended=full,vip`) for basically every request where user information is in some form (like in [List Likes - Get all users who liked a list](https://trakt.docs.apiary.io/#reference/lists/list-likes/get-all-users-who-liked-a-list)) returned.
But it is not or in some cases not documented.
So, is this just wrong API behavior or just missing documentation?

**Comment by rectifyer** [comment, marked as answer]
This is correct behavior. The API docs discuss in these sections, with example code.

https://trakt.docs.apiary.io/#introduction/extended-info
https://trakt.docs.apiary.io/#reference/users/profile/get-user-profile
